tools/types/build.js

```diff
@@ -14,7 +14,9 @@
  * limitations under the License.
  */
 
-const chromeTypesRepoURL = 'https://unpkg.com/chrome-types@latest/';
+// FIXME: remove before submit
```

Comment:
Should this be removed?

Reply:
Yup, I will. This was for testing because the contents of this file have changed but I've not yet marked the new version at "latest".

This should be the last time we have to do this.